README.md

```diff
@@ -33,20 +34,17 @@ E.g. `ca_quebec.*` implies Canada.
 
 The [repository's](http://download.xcsoar.org/repository) `update` field is generated from the git commit date.
 
-
-### Output
+## Output
 
 The following manifest files are built by, with, and from, this repo (required by ...):
 
-1. https://download.xcsoar.org/repository (XCSoar)
-1. https://download.xcsoar.org/waypoints/waypoints.js (website)
-1. https://download.xcsoar.org/waypoints/waypoints_compact.js (website)
-1. https://download.xcsoar.org/maps/maps.config.js (website & `mapgen`)
-1. https://download.xcsoar.org/waypoints/xcsoar_waypoints.cup (`mapgen`)
+1.   [https://download.xcsoar.org/repository](https://download.xcsoar.org/repository) (XCSoar)
+2.   [https://download.xcsoar.org/waypoints/waypoints.js](https://download.xcsoar.org/waypoints/waypoints.js) (website)
+3.   [https://download.xcsoar.org/waypoints/waypoints_compact.js](https://download.xcsoar.org/waypoints/waypoints_compact.js) (website)
+4.   [https://download.xcsoar.org/maps/maps.config.js](https://download.xcsoar.org/maps/maps.config.js) (website & `mapgen`)
+5.   [https://download.xcsoar.org/waypoints/xcsoar_waypoints.cup](https://download.xcsoar.org/waypoints/xcsoar_waypoints.cup) (`mapgen`)
```

Comment:
Is there a reason to be redundant with the URL here? If we ditch the `[]()` I believe the URLs will still be links in the end, at least this is the case with https://soaringweb.org/ above.

Reply:
Also, the bullet points are indented one space too much.

Reply:
I just wanted it to work with any markdown parser. 
but i found a better [official way](https://www.markdownguide.org/basic-syntax/#formatting-links) here.